Project4/style.css

```
@@ -0,0 +1,637 @@
+    outline: 0;
+}
+
+.register-button {
+    display: block;
+    width: 100%;
+    height: 42px;
+    margin-top: 25px;
+    font-size: 16px;
+    font-weight: bold;
+    color: #494d59;
+    text-align: center;
+    text-shadow: 0 1px rgba(255, 255, 255, 0.5);
+    background: #fcfcfc;
+    border: 1px solid;
+    border-color: #d8d8d8 #d1d1d1 #c3c3c3;
+    border-radius: 2px;
+    cursor: pointer;
+    -webkit-box-shadow: inset 0 -1px rgba(0, 0, 0, 0.03), 0 1px rgba(0, 0, 0, 0.04);
+    box-shadow: inset 0 -1px rgba(0, 0, 0, 0.03), 0 1px rgba(0, 0, 0, 0.04);
+}
+
+.register-button:active {
+    background-image: -webkit-linear-gradient(top, #eeeeee, #fcfcfc);
+    background-image: -moz-linear-gradient(top, #eeeeee, #fcfcfc);
+    background-image: -o-linear-gradient(top, #eeeeee, #fcfcfc);
+    background-image: linear-gradient(to bottom, #eeeeee, #fcfcfc);
+    -webkit-box-shadow: inset 0 1px rgba(0, 0, 0, 0.03);
+    box-shadow: inset 0 1px rgba(0, 0, 0, 0.03);
+}
+
+.register-button:focus {
+    outline: 0;
+}
+
+.register-switch {
+    height: 32px;
+    margin-bottom: 15px;
+    padding: 4px;
+    background: #6db244;
+    border-radius: 2px;
+    background-image: -webkit-linear-gradient(top, #60a83a, #7dbe52);
+    background-image: -moz-linear-gradient(top, #60a83a, #7dbe52);
+    background-image: -o-linear-gradient(top, #60a83a, #7dbe52);
+    background-image: linear-gradient(to bottom, #60a83a, #7dbe52);
+    -webkit-box-shadow: inset 0 1px rgba(0, 0, 0, 0.05), inset 1px 0 rgba(0, 0, 0, 0.02), inset -1px 0 rgba(0, 0, 0, 0.02);
+    box-shadow: inset 0 1px rgba(0, 0, 0, 0.05), inset 1px 0 rgba(0, 0, 0, 0.02), inset -1px 0 rgba(0, 0, 0, 0.02);
+}
+
+.register-switch-input {
+    display: none;
+}
+
+.register-switch-label {
+    float: left;
+    width: 50%;
+    line-height: 32px;
+    color: white;
+    text-align: center;
+    text-shadow: 0 -1px rgba(0, 0, 0, 0.2);
+    cursor: pointer;
+}
+
+.register-switch-input:checked+.register-switch-label {
+    font-weight: 500;
+    color: #434248;
+    text-shadow: 0 1px rgba(255, 255, 255, 0.5);
+    border-radius: 2px;
+    -webkit-box-shadow: 0 1px 1px rgba(0, 0, 0, 0.15), 0 0 0 1px rgba(0, 0, 0, 0.1);
+    box-shadow: 0 1px 1px rgba(0, 0, 0, 0.15), 0 0 0 1px rgba(0, 0, 0, 0.1);
+}
+
+:-moz-placeholder {
+    color: #aaa;
+    font-weight: 300;
+}
+
+::-moz-placeholder {
+    color: #aaa;
+    font-weight: 300;
+    opacity: 1;
+}
+
+::-webkit-input-placeholder {
+    color: #aaa;
+    font-weight: 300;
+}
+
+:-ms-input-placeholder {
+    color: #aaa;
+    font-weight: 300;
+}
+
+::-moz-focus-inner {
+    border: 0;
+    padding: 0;
+}
+
+
+h1 {
+    font-family: 'Orbitron', sans-serif;
+
+    font-size: 45px;
+}
+
+h3 {
+    font-family: 'Gugi', cursive;
+    font-size: 50px;
+    color: #f0ebeb;
+}
+
+h4 {
+    font-family: 'Teko', sans-serif;
+}
+
+p {
+    font-family: 'Khand',
+        sans-serif;
+    font-size: 20px;
+    color: #cccccc;
+
+}
+
+.mt medium diagonal {
```

This selector is for this element in your HTML:
```html
        <button class="mt medium diagonal">Ticket</button>
```
and to properly target it with CSS, you need the selector to be:
```suggestion
.mt.medium.diagonal {
```
This targets an element where the classnames are "mt" "medium" _and_ "diagonal." You may not need all three classnames, however.